src/client/app.js

```diff
@@ -9,7 +9,7 @@ import Bacon from 'baconjs'
 import _ from 'lodash'
 import {COG, HDG, MAX_ZOOM, MIN_ZOOM, EXTENSION_LINE_OFF, EXTENSION_LINE_2_MIN, EXTENSION_LINE_5_MIN, EXTENSION_LINE_10_MIN} from './enums'
 import Map from './map'
-import Connection from './data-connection'
```

Comment:
Refactoring code from of the master branch inside a feature branch should be avoided.  It's a PITA when it come to merge multiple branches. Refactoring process are bumpy enough to worth it's own branch :D.